tsa/constants.go

```diff
@@ -0,0 +1,8 @@
+package tsa
+
+// CIS recommends a certain set of MAC algorithms to be used in SSH connections. This restricts the set from a more permissive set used by default by Go.
+// See https://infosec.mozilla.org/guidelines/openssh.html and https://www.cisecurity.org/cis-benchmarks/
+var AllowedMACs = []string{
+	"hmac-sha2-256-etm@openssh.com",
+	"hmac-sha2-256",
```

Comment:
Why only allow these two MAC's? What about the `512`'s?
From the link in the comment, this is mozilla's list:
```
hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com
```

Reply:
Go doesn't support it.......

Reply:
https://github.com/golang/crypto/blob/d864b10871cd4370fe574816b489c819c675ccc7/ssh/common.go#L69

Reply:
@taylorsil

Reply:
Annoying that there are constants for cipher suites but not MACs 😐

Reply:
They're also `string` instead of tls's `uint16` arrays. But wait, SSH also has Ciphers (different from TLS cipher suites apparently) and those are a mix of hard coded strings and constants! https://github.com/golang/crypto/blob/d864b10871cd4370fe574816b489c819c675ccc7/ssh/common.go#L31

Oh did I mention that the Key Exchange Algorithms and Host Key Algorithms that are also part of SSH uses constants?

It's a big beautiful mess of things ¯\\_(ツ)_/¯